app/models/local_census_record.rb

```diff
@@ -0,0 +1,3 @@
+class LocalCensusRecord < ActiveRecord::Base
+
```

Comment:
🤔  its a feature not defined yet.... Yes, forks may use it in different ways, either with a /admin interface to upload a .csv or .xls file (that would go through Rails), but someone a bit hacky could like to load a sql dump directly into postgresql :D.

Anyway for those that go through Rails the validations may be a good idea. I think on madrid we'll do that, even here in consul, just because this "Local Census" may be used for Demos as well (not only for Associations).